spec/requests/api/v2/providers/courses/create_spec.rb

```diff
@@ -10,26 +10,36 @@
   end
   let(:provider) { create(:provider, organisations: [organisation]) }
   let(:recruitment_cycle) { provider.recruitment_cycle }
+  let(:primary_with_mathematics) { find_or_create(:subject, :primary_with_mathematics) }
   let(:course) { build(:course, provider: provider) }
   let(:jsonapi_renderer) { JSONAPI::Serializable::Renderer.new }
   let(:create_path) do
     "/api/v2/recruitment_cycles/#{recruitment_cycle.year}" +
       "/providers/#{course.provider.provider_code}/courses"
   end
 
+  let(:updated_subjects) do
+    {
+      subjects: {
+        data: [{ "type" => "subject", "id" => primary_with_mathematics.id.to_s }],
+      },
+    }
+  end
+
   def perform_request(course)
     jsonapi_data = jsonapi_renderer.render(
       course,
       class: {
         Course: API::V2::SerializableCourse,
       },
     )
+    jsonapi_data[:data][:relationships] = updated_subjects
```

Comment:
As discussed IRL, can we turn this around and do the `includes` in the `render` above, and just not use the `included` part of `jsonapi_data`?

The below would then become:

```ruby
    post create_path,
         headers: { "HTTP_AUTHORIZATION" => credentials },
         params: {
           _jsonapi: { data: jsonapi_data[:data] }
         }
```